src/openforms/prefill/base.py

```diff
@@ -18,7 +18,7 @@ class BasePlugin(AbstractBasePlugin):
     requires_auth: AuthAttribute | None = None
     for_components: Container[str] = AllComponentTypes()
 
-    def get_available_attributes(self) -> Iterable[Tuple[str, str]]:
+    def get_available_attributes(self) -> Iterable[tuple[str, str]]:
```

Comment:
this clutters the diff, only make those changes/cleanups if the line is already affected anyway (due to changes in method signature for example)